api/src/pcapi/routes/serialization/thumbnails_serialize.py

```diff
@@ -38,9 +38,6 @@ def get_image_as_bytes(self, request) -> bytes:
             image_as_bytes = blob.read()
             return validation.get_uploaded_image(image_as_bytes)
 
-        if self.thumb_url:
-            return validation.get_distant_image(self.thumb_url)
-
         raise validation.exceptions.MissingImage
```

Comment:
(détail, au passage :) )

```suggestion
        raise validation.exceptions.MissingImage()
```